[BREAKING] Feature: Simplify and improve the consumer (#11753)
paperless-ngx/paperless-ngx: docs/troubleshooting.md

If you notice that the consumer will only pickup files in the
consumption directory at startup, but won't find any other files added
later, you will need to enable filesystem polling with the configuration
option [`PAPERLESS_CONSUMER_POLLING_INTERVAL`](configuration.md#PAPERLESS_CONSUMER_POLLING_INTERVAL).

> This will disable listening to filesystem changes with inotify and

Reviewer: Its not inotify any more right?

```suggestion
This will disable automatic listening for filesystem changes and
```
